lib/content._coffee

```diff
@@ -47,11 +44,14 @@ getFilePathForId = (id) ->
     id = id.replace /([A-Z])/g, '_$1'
     Path.join DIR_BY_ID_PATH, "#{id}.json"
 
-getFilePathForURL = (url) ->
-    if idFile = DATABASE["#{hashURL url}.json"]
-        Path.join DIR_BY_ID_PATH, idFile.replace '../content-by-id', ''
-    else
-        ''
+readFileForURL = (url, _) ->
+    cloudClient.download
+        container: 'content/content-by-url'
+        remote: "#{hashURL url}.txt"
```

Comment:
I believe `content-by-url` needs to be part of the path (`remote`). I think `container` is just like S3 bucket.

Might also be good to make this a constant or config, in case we want to separate production from development.